MatrixSDKTests/MXAccountDataTests.m

```diff
@@ -93,24 +93,27 @@ - (void)testIgnoreUsersStorage
                     if (notif.object == bobSession)
                     {
                         // Yield so that bobSession completes the saving to the cache.
+                        // TODO: Find a more accurate point of sync...
                         dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
It needs 2 delays to make the test pass :/
The 1st `dispatch_async` was already a workaround to manage `Yield so that bobSession completes the saving to the cache`.

As there is a new additional async step in the code processing the initial `/sync`, we need an additional delay. So, I stupidly duplicate the already existing delay.

There is probably  a better way for synchronising this, but this is code for tests. The main thing to test here is the data validity, not the timing.

Reply:
Ok otherwise you can use `dispatch_after` to add a sufficient delay.

Reply:
Why not but 2 `dispatch_asyncs`  appear more accurate for the moment than a single `dispatch_after` with a random time value.